### DISCUSSION TITLE
Python proyecto

### DISCUSSION THREAD

**Discussion by HidoGG** [CodeSystem2022/Team-Fortran-2023#82] [Ideas]
Propongan ideas para ir pensando que vamos a hacer para **el Proyecto de Python**.

**Comment by LucasVizcaino** [comment]
Podría ser un sistema de gestión de tareas, una aplicación que permita a los usuarios crear, organizar y realizar un seguimiento de sus tareas o actividades y tendría funciones como asignación de tareas, recordatorios, fechas límite y generación de informes.
o alguna otra app con python
